### DISCUSSION TITLE
Node syncing - wallet is not syncing - what to do?

### DISCUSSION THREAD

**Discussion by chrismikefowler1** [Chia-Network/chia-blockchain#5336] [Support]
**Describe the bug**
Wallet wont sync, the node has fully synced. I am unable to farm and i have over 100 plots!!! what do i do?

click on the wallet page.  My number is stuck at 66501, when my node is up to date - but also says not synced(thought its synced).  Ive tried reinsatalling. I've port forwarded. Ive reinstalled chia.  ive done everything ive seen online.  what can i do? Its been going on for me for 3 days.

**Comment by FSys77** [comment]
> Wallet wont sync, the node has fully synced. I am unable to farm and i have over 100 plots!!! what do i do?

Relax :-)

With your meager 100 plots your chance to win a block in the next 7 days is 2%

**Comment by chrismikefowler1** [comment]
well i dont know about that - thats good odds if true. but i'm trying to sync...is everything having this issue? i have 1 computer left thats staying synced.

**Comment by Code-Josh** [comment]
I had the Problem too, but my solution was to create Plots, i dont know why but the wallet doesnt sync if you don't have any plots on your System. Maybe try to create new Plots or to delete the .chia Folder

**Comment by chrismikefowler1** [comment]
i deleted the .chia folder and started over syncing.  it did not help - because not i'm at full sync on the node and again stuck at the same spot on the wallet

**Comment by chrismikefowler1** [comment]
i created a new wallet and its starting to sync - good thing i'm a baby

**Comment by casa10** [comment]
Hello,
More and more have same problem: lost synced suddendly without reason. After, difficult to synced again.
Technical support here are often same problem described.
Just hope they will find solution to this problem, because a lot can't farm well.

**Reply by pluto2021** [reply]
Not sure if there is an architectural issue in the blockchain. When the blockchain is not as large, you could easily fix the syncing issue by deleting the .chia folder. Once the blockchain gets huge, once some peers start to fall out due to issues like windows update that would kill the Chia process, power outage, trying to stop plotting within GUI, etc., maybe more and more peers would not be able to sync due to corrupted local chain or block when those events happened. If you are connected to peers that have trouble syncing, e.g. with a broken or corrupted chain, the sync may stop and get stuck there. You may get synced to peers with slow syncing, or even got synced to a corrupted chain that may never move! This syncing issue could spread like a virus.  In addition, as the network is growing exponentially, so does the block chain, while syncing is in general linear with a cap. So once the block chain gets large enough, some may never be able to catch of speed of blockchain growth. The developer team needs to figure out a more reliable way of keep things synced regardless of what happens. It seems that the blockchain has been designed assuming no anomalies. In real world, this would not be a good strategy. The blockchain needs to be robust enough. It also seems that the main net was released prematurely.  The current block chain might need a hard fork to transition to another more robust chain.

**Reply by Tibi38** [reply]
my problem by missing sync is that there is no connection more to other peers. even if i enter one by hand it not connect. After restarting the application on windows, it sync again for some hours. it looks not simular to your description

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
